packages/sources/src/components/accordion/accordion.args.ts

```diff
@@ -0,0 +1,37 @@
+import { ArgTypes } from '../../stories-helpers';
+
+import { Accordion, AccordionSection, AccordionVariant } from './accordion.models';
+
+export interface AccordionArgs {
+  variant?: AccordionVariant;
+  sections: AccordionSection[];
+  reverseAlign?: boolean;
+}
+
+export const accordionArgTypes: ArgTypes<AccordionArgs> = {
+  variant: {
+    options: [undefined, 'dso-accordion-compact', 'dso-accordion-conclusion'],
+    control: {
+      type: 'select',
+      labels: {undefined: 'default', 'dso-accordion-compact': 'compact', 'dso-accordion-conclusion': 'conclusion'}
+    }
+  },
+  sections: {
+    control: {
+      disable: true
+    }
+  },
+  reverseAlign: {
+    control: {
+      disable: true
```

Comment:
Waarom is `reverseAlign` disabled?

Reply:
Goeie vraag, `reverseAlign` lijkt handig te zijn om als control te kunnen schakelen, maar in het geval van de addons zal dit mis gaan. We ondersteunen niet alle soorten addons in de `reverseAlign` variant (dit is besproken met Bram), het zal dus mis gaan bij de warnings en de attachments en ik denk niet dat we dat willen showcasen.

De enige variant die ondersteunt gaat worden is te zien in de `alignment` variant

Reply:
OK, we laten het voor nu even